pkgs/by-name/mo/modrinth-app-unwrapped/package.nix

```diff
@@ -3,51 +3,53 @@
   stdenv,
   fetchFromGitHub,
   rustPlatform,
-  buildGoModule,
+  cacert,
   cargo-tauri,
   darwin,
   desktop-file-utils,
-  esbuild,
   libsoup,
-  nix-update-script,
   nodejs,
   openssl,
   pkg-config,
-  pnpm_8,
+  pnpm_9,
+  turbo,
   webkitgtk,
 }:
 
+let
+  pnpm = pnpm_9;
+in
 rustPlatform.buildRustPackage rec {
   pname = "modrinth-app-unwrapped";
-  version = "0.7.1";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner = "modrinth";
     repo = "code";
-    rev = "refs/tags/v${version}";
-    sha256 = "sha256-JWR0e2vOBvOLosr22Oo2mAlR0KAhL+261RRybhNctlM=";
+    rev = "a0bd011b808cdc998ef27960f610a8d99550c914";
+    sha256 = "sha256-zpFJq7if5gOx7jvwpE73lqH4Vpif0MJMPIGsgtThKVk=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "tauri-plugin-single-instance-0.0.0" = "sha256-Mf2/cnKotd751ZcSHfiSLNe2nxBfo4dMBdoCwQhe7yI=";
+      "sqlx-0.8.0-alpha.0" = "sha256-M1bumCMTzgDcQlgSYB6ypPp794e35ZSQCLzkbrFV4qY=";
+      "tauri-plugin-single-instance-0.0.0" = "sha256-DZWTO2/LevbQJCJbeHbTc2rhesV3bNrs+BoYm2eMakA=";
     };
   };
 
-  pnpmDeps = pnpm_8.fetchDeps {
+  pnpmDeps = pnpm.fetchDeps {
     inherit pname version src;
-    sourceRoot = "${src.name}/theseus_gui";
-    hash = "sha256-g/uUGfC9TQh0LE8ed51oFY17FySoeTvfaeEpzpNeMao=";
+    hash = "sha256-xD04CD3fevIKZoXpjMo3tRF2r7K2IiMh1cNS7/FNx2I=";
   };
-  pnpmRoot = "theseus_gui";
 
   nativeBuildInputs = [
+    cacert
```

Comment:
Wasn't this removed a couple of commits ago?

Reply:
It was as previously pnpm required it to fetch dependencies, but fetchPnpmDeps takes care of that now
The reintroduction is because of 0.8.2 now using turbo, which also requires cacert at runtime (this is commented now)